package.json

```
@@ -128,7 +128,7 @@
     "enzyme-to-json": "^3.3.5",
     "eslint": "^7.19.0",
     "html-webpack-plugin": "^4.3.0",
-    "jest": "^26.0.1",
+    "jest": "^27.5.1",
```

Can we format it using only the major version? It's the same syntax and helps separate it from packages we locked to specific versions. It'll also make upgrading easier by allowing us to bump up the number.
```suggestion
    "jest": "27",
```

---

Sure. [f01704f](https://github.com/cerner/terra-toolkit/pull/829/commits/f01704fa0ec5f23e71006be6eaa97d7c2b766fa0)